tests/test_api_hosts_get.py

```diff
@@ -2103,3 +2103,39 @@ def test_get_host_from_different_org(mocker, api_get):
     url = build_hosts_url()
     response_status, _ = api_get(url)
     assert_response_status(response_status, 403)
+
+
+def test_query_by_staleness_using_colums(db_create_multiple_hosts, api_get, subtests):
+    with (
+        patch("app.staleness_serialization.get_flag_value", return_value=True),
+        patch("app.models.get_flag_value", return_value=True),
+        patch("app.serialization.get_flag_value", return_value=True),
+        patch("api.host_query_db.get_flag_value", return_value=True),
+    ):
```

Comment:
`patch()` doesn't return a context manager, so you can remove the `with`. I've done this same thing in the past too, and am trying to skip it going forward
```suggestion
    patch("app.staleness_serialization.get_flag_value", return_value=True)
    patch("app.models.get_flag_value", return_value=True)
    patch("app.serialization.get_flag_value", return_value=True)
    patch("api.host_query_db.get_flag_value", return_value=True)
```